src/inspector_agent.cc

```diff
@@ -635,6 +678,14 @@ void AgentImpl::WorkerRunIO() {
   err = uv_async_init(&child_loop_, &io_thread_req_, AgentImpl::WriteCbIO);
   CHECK_EQ(err, 0);
   io_thread_req_.data = this;
+  uv_fs_t* req = new uv_fs_t();
+  if (!script_name_.empty()) {
+    req->data = this;
+    if (uv_fs_realpath(&child_loop_, req, script_name_.c_str(),
+        AgentImpl::SetAbsolutePath)) {
```

Comment:
I think it would be okay for simplicity to omit the callback and execute uv_fs_realpath() synchronously.  This is hardly a critical code path.